src/config.ts

```diff
@@ -405,23 +405,23 @@ export const GAS_SCHEDULE_V1: FeeSchedule = {
     [ERC20BridgeSource.MultiBridge]: () => 3.5e5,
     [ERC20BridgeSource.UniswapV2]: fillData => {
         // TODO: Different base cost if to/from ETH.
-        let gas = 100e3;
+        let gas = 90e3;
         const path = (fillData as UniswapV2FillData).tokenAddressPath;
         if (path.length > 2) {
-            gas += (path.length - 2) * 50e3; // +50k for each hop.
+            gas += (path.length - 2) * 60e3; // +60k for each hop.
         }
         return gas;
     },
     [ERC20BridgeSource.SushiSwap]: fillData => {
         // TODO: Different base cost if to/from ETH.
-        let gas = 105e3;
+        let gas = 95e3;
         const path = (fillData as UniswapV2FillData).tokenAddressPath;
         if (path.length > 2) {
-            gas += (path.length - 2) * 50e3; // +50k for each hop.
+            gas += (path.length - 2) * 60e3; // +60k for each hop.
         }
         return gas;
     },
-    [ERC20BridgeSource.Balancer]: () => 1.5e5,
+    [ERC20BridgeSource.Balancer]: () => 1.2e5,
```

Comment:
150k seems high, see this Balancer [ETH-YFI trade](https://dashboard.tenderly.co/0x-org/exchange-proxy/tx/main/0x2906c197b6e11c8587c772f25de1586a76ed520820a6e02500df235abd620fc5/gas-usage)